compose/release-notes.md

```diff
@@ -8,6 +8,51 @@ redirect_from:
 ---
 
 
+## 2.6.1
+
+(2022-06-23)
+
+
+### Bug Fixes
+
+- Added support for setting secrets from environment variable. Fixes [#251](https://github.com/compose-spec/compose-spec/issues/251){:target="_blank" rel="noopener" class="_"}.
```

Comment:
```suggestion
- Added support for setting secrets from environment variable. See [compose-spec/compose-spec#251](https://github.com/compose-spec/compose-spec/issues/251){:target="_blank" rel="noopener" class="_"}.
```
This is a feature rather than a fix, so it might actually make more sense for it to be separate from this section. But for consistency, I think specifying the repo (similar to `docker/for-mac#xxxx` below) is also nice since it's linking to something outside the main Compose repo

Reply:
OK, one small question would "Added links to container create request" not be an enhancement at the same level?